### ISSUE TITLE
Test case 10B : "Error in FHIR validation"

### ISSUE THREAD

**Issue by sebwautelet** [hl7-be/lab#214]
I submitted a file for test case 10B (incoming messages ID 847 & 848) but its status comes out as "Error in FHIR validation", and they don't show up in "My test results". The file was however succesfully validated on https://fhir-testserver.be/index.php/visualization_webapp
Can you check what's wrong?

**Comment by bdc-ehealth** [comment]
@sebwautelet ,
Indeed, you are right, there are no problems with the validation. We will investigate.

**Comment by bdc-ehealth** [comment]
@sebwautelet ,

We were able to trace this problem to a transient issue on the international HL7 terminology server. The problem has disappeared, and we hope it stays so. You can retry to send the files.

**Comment by sebwautelet** [comment]
Thanks @bdc-ehealth , our example has now been processed (test result #176). It didn't pass because of this rule :
`Bundle.entry.where(resource is Observation).resource.where(code.coding.where(system = 'http://loinc.org' and code = '66121-5')).exists()`
So I have some questions :

- could this LOINC code (or LP7839-6?) be set on the DiagnosticReport instead? Our Observation linked to the DiagnosticReport refers to a more specific procedure (immunofluorescence), not sure if there's a LOINC code for it though.
- we have more sub-Observations for the different paragraphs of the pathology report (macroscopy, microscopy, conclusion...). Is this OK or should all the text be contained in the same Observation?

**Comment by sebwautelet** [comment]
Hello @bdc-ehealth , do you have an update on my 2 questions above?
Thanks

**Comment by bdc-ehealth** [comment]
@sebwautelet ,

Sorry for the late reply, your message got out of my sight. 
1) This is not my area of expertise. I will have to ask a clinical biologist.
2) Pathology has not been fully structured yet, so you have the choice: either you create a tree of observations if they fit within the current agreements, or you take only one observation and put everything in the string value of the observation. If you take the first solution, you of course have the risk that if pathology is structured in the future, the structure can be different from the one that you are creating now. BTW. If you have ideas how to structure pathology in FHIR, you can always give them as an input to the HL7 FHIR workgroup about Lab Reports.
